python/observability-online-eval/eleven-labs/stt_tts_pipeline.py

```
@@ -0,0 +1,189 @@
+
+from maxim import Maxim
+from maxim.logger.components.trace import TraceConfigDict
+from maxim.logger.elevenlabs import instrument_elevenlabs
+
+load_dotenv()
+
+# Configuration
+ELEVENLABS_API_KEY = os.getenv("EL_API_KEY")
+
+if not ELEVENLABS_API_KEY:
+    raise ValueError("ELEVENLABS_API_KEY environment variable is not set")
```

Ruff (0.14.5)

Avoid specifying long messages outside the exception class

(TRY003)